README.md

```diff
@@ -32,7 +34,9 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 ----
 
 *Code is licensed under the [Apache License, Version 2.0](/LICENSE).*  
+
 *Documentation/examples are licensed under [Creative Commons BY-SA 4.0](/docs/LICENSE).*  
+
```

Comment:
These whitespace lines are too much. Now it's a new line and an empty line.

Reply:
I just added a \n at the end of each license line. So I think it was working correctly before. Will remove those \n.

Reply:
Yeah I saw that, but you already regenerated the README so this "bug" should have been showing up before. Not sure, why it came up now.

Reply:
It's kinda weird. because even without \n it works fine and each license is on a separate line.

Reply:
Weird, but as long as the actual bug is fixed \o/